src/modules/fw_pos_control_l1/FixedwingPositionControl.cpp

```diff
@@ -888,8 +888,6 @@ FixedwingPositionControl::control_auto(const float control_interval, const Vecto
 {
 	update_in_air_states(now);
 
-	_hdg_hold_yaw = _yaw;
```

Comment:
Just to verify, presumably this was done (lazily) so that `_hdg_hold_yaw` is set if you transition back to manual. Is that handled explicitly?

Reply:
I believe this line covers it, but if you see a corner case i missed, definitely let me know.

https://github.com/PX4/PX4-Autopilot/blob/e30b0aae134e3317c8669d7fb851d750aaf93596/src/modules/fw_pos_control_l1/FixedwingPositionControl.cpp#L817